src/System/Components/RouterLink.tsx

```diff
@@ -121,7 +122,6 @@ const VALID_ROUTER_LINK_PROPS = [
   "data-test",
   "data-testid",
   "exact",
-  "onClick",
```

Comment:
Looks like most of these can be removed as well when using `isPropValid` - https://github.com/emotion-js/emotion/blob/main/packages/is-prop-valid/src/index.ts includes `children`, `style`, `target`, etc.

Reply:
Looks like everything except `activeClassName`, `activeStyle`, `exact` — given that it blanket includes [`on*`](https://github.com/emotion-js/emotion/blob/e310c6e6a9f0b4f55030831ef65cd07c7105ae01/packages/is-prop-valid/src/index.ts#L12-L13) and [`data*` and `aria*`](https://github.com/emotion-js/emotion/blob/e310c6e6a9f0b4f55030831ef65cd07c7105ae01/packages/is-prop-valid/src/props.js#L499)